content/fair/interoperable/index.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Galaxy is Interoperable
+highlight: true
+---
+
+Galaxy as both a project and a platform facilitates FAIR research, addressing common needs, invoking available standards and contributing to several upstream projects. 
+
+This page focusses on FAIR's 'I' component, so highlighting interoperability measures, pointing to respective implementations and answering a range of questions around the topic. Contents have been generated from [Galaxy Europe's successful evaluation](/news/2023-12-14-ELIXIR-RIR-for-Galaxy-Europe/) of being an [ELIXIR 'Recommended Interoperability Resource' (RIR)](https://elixir-europe.org/platforms/interoperability/rirs), for which central considerations have been addressed. Both asked questions and their answers should be shared here with the community, hopefully serving as references for future needs. 
+
+## Resource facilitation to scientific research
+
+### Brief description of the **function** of Galaxy as an interoperability resource
+
+Galaxy is an interoperable open-source platform for scientific data analysis and sharing, covering diverse research fields such as *omics, machine learning, and climate science. The platform emphasises transparency, reproducibility, and reusability, facilitating sharing of tools, workflows, visualisations, and data, while capturing all provenance information. Galaxy also offers strong research data management tools, covering data import, organisation, annotation, sharing and export. With the aim of accelerating scientific discovery and innovation, the platform encourages researchers to share their data and analysis workflows with the wider scientific community. Galaxy provides a borderless and efficient approach to support non-technical users in effective research.
+
+### Describe the **scope**, and the users of the resource. How is Galaxy positioned with respect to other similar interoperability resources?
+
+The usegalaxy.* servers (in the US, EU, AUS) have an extensive user base each of tens of thousands of registered users, adding a few thousand new users every month. These users come from different scientific fields and are majorly non-technical scientists, who intend to answer research questions using data science methods. The majority of users is interested in analysing own data in the context of publicly available reference resources and data collections. Their main goal is to develop workflows of tools and to apply them on a large scale. According to this user profile, the Galaxy Training Network (GTN, https://training.galaxyproject.org) has developed broad and interactive high-quality training materials, being deeply rooted in both Galaxy’s community and technical infrastructure.
```

Comment:
The usegalaxy.* servers (in the US, EU, AUS) have an extensive user base each of tens of thousands of registered users, adding a few thousand new users every month. These users come from different scientific fields and are the majority do not have a background in data-intensive methodologies, who intend to answer research questions using data science methods. The majority of users are interested in analysing their own data in the context of publicly available reference resources and data collections. Their main goal is to develop workflows of tools and to apply them on a large scale. According to this user profile, the Galaxy Training Network (GTN, https://training.galaxyproject.org) has developed broad and interactive high-quality training materials, being deeply rooted in both Galaxy’s community and technical infrastructure.